packages/devtools_app/lib/src/service/isolate_state.dart

```diff
@@ -13,25 +11,25 @@ import 'package:vm_service/vm_service.dart' hide Error;
 class IsolateState {
   IsolateState(this.isolateRef);
 
-  ValueListenable<bool> get isPaused => _isPaused;
+  ValueListenable<bool?> get isPaused => _isPaused;
 
   final IsolateRef isolateRef;
 
-  Future<Isolate> get isolate => _isolate.future;
-  Completer<Isolate> _isolate = Completer();
+  Future<Isolate> get isolate => _completer.future.then((value) => value!);
```

Comment:
_completer may complete with null, so we do not want to add a !. Make the return type `Future<Isolate?>`

Reply:
done